framework/Models/EnsembleModel.py

```diff
@@ -287,6 +287,7 @@ def initialize(self,runInfo,inputs,initDict=None):
       for inFile, used in checkDictInputsUsage.items():
         if not used:
           unusedFiles+= " "+inFile.name
+      # Why is this an error? It breaks runs for the sake of a placeholder.
```

Comment:
because it is to avoid to have users to think that some inputs are used even if are not (expectation management)

Reply:
Hm. I guess this is slightly the opposite of the other entities in RAVEN, where you can define extra ones that don't get used. Obviously this doesn't need changing now, it just felt somewhat out-of-place.

Reply:
comment removed